src/EntityStore.ts

```diff
@@ -14,22 +22,45 @@ export default class EntityStore {
     );
   }
 
+  /**
+   * The loading state of the store. If it's true, the store is fetching data.
+   */
   loading = false;
+  /**
+   * The data that represents an entity.
+   */
   data: unknown | null = null;
```

Comment:
couldn't typing be dynamic by generics?

Reply:
in update function we have a patch request with a `this.setData` call using `response.data` that is `unknown`
if i put a `T` in Axios response we are assuming that the patch or put endpoint will respond with the same entity structure of the GET one

Reply:
can i proceed?

Reply:
Yes, PR confirmed